projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeployJob.tsx

```diff
@@ -79,52 +67,46 @@ export default class DeployJobDialog extends Component<IDeployJobDialogProps> {
             }
         }
     }
-    /**
-  * Callback invoked upon changing the job name input.
-  *
-  * @param event - event object
-  */
-    private onNameChange = (event: any): void => {
-        this.populateJobData(VdkOption.NAME, event.currentTarget as HTMLInputElement);
-    };
-    /**
-   * Callback invoked upon changing the job team input.
-   *
-   * @param event - event object
-   */
-    private onTeamChange = (event: any): void => {
-        this.populateJobData(VdkOption.TEAM, event.currentTarget as HTMLInputElement);
-    };
-    /**
-   * Callback invoked upon changing the job rest-api-url input.
-   *
-   * @param event - event object
-   */
-    private onRestApiUrlChange = (event: any): void => {
-        this.populateJobData(VdkOption.REST_API_URL, event.currentTarget as HTMLInputElement);
-    };
-    /**
-   * Callback invoked upon changing the job path input.
-   *
-   * @param event - event object
-   */
-    private onPathChange = (event: any): void => {
-        this.populateJobData(VdkOption.PATH, event.currentTarget as HTMLInputElement);
-    };
-    /**
-  * Callback invoked upon changing the deployent reason input.
-  *
-  * @param event - event object
-  */
-    private onDeploymentReasonChange = (event: any): void => {
-        this.populateJobData(VdkOption.DEPLOYMENT_REASON, event.currentTarget as HTMLInputElement);
-    };
-    /**
-  *  Helper function for populating jobData
-  */
-    private populateJobData(option: VdkOption, input: HTMLInputElement) {
-        let value = input.value;
-        if (!value) value = "";
-        jobData.set(option, value);
-    }
 }
+
+export async function showCreateDeploymentDialog() {
+    const result = await showDialog({
+      title: 'Create Deployment',
+      body: (
+        <DeployJobDialog jobName='job-to-deploy' jobPath={sessionStorage.getItem('current-path')!} jobTeam='my-team'></DeployJobDialog>
+      ),
+      buttons: [Dialog.okButton(), Dialog.cancelButton()]
```

Comment:
nit: aren't those buttons always, or most of the time the same? maybe default to the most frequent usage?

Reply:
I cannot understand what you mean like that, can u clarify?

Reply:
resolving it now, we can discuss it with the later changes